ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -265,6 +265,65 @@ presubmits:
           requests:
             cpu: 10m
     trigger: (?m)^/test( | .* )boskos-config-generation,?($|\s.*)
+  - agent: kubernetes
```

Comment:
@jtaleric Do we need this change?
May be my approve did not go through because of this?

Reply:
let's try rebase